checkup-bot-backend/src/main/java/com/checkupbot/checkupbotbackend/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package com.checkupbot.checkupbotbackend.controllers;
+
+import com.checkupbot.checkupbotbackend.requests.AuthenticationRequest;
+import com.checkupbot.checkupbotbackend.responses.AuthenticationResponse;
+import com.checkupbot.checkupbotbackend.services.AuthenticationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/auth")
```

Comment:
In Nginx we already have api in front of the path so we can remove '/api' because otherwise we have to ping '/api/api...'

Reply:
Also since we only have one auth method '/auth/authenticate' seems a bit repetitive

Reply:
but I'm being picky